CHANGELOG.md

```diff
@@ -1,147 +1,202 @@
+
+- [#3149](https://github.com/mochajs/mocha/issues/3149): **Drop Node.js v4.x support** ([**@outsideris**](https://github.com/outsideris))
+- [#3556](https://github.com/mochajs/mocha/issues/3556): Changes to command-line options ([**@boneskull**](https://github.com/boneskull)):
+  - `--grep` and `--fgrep` are now mutually exclusive; attempting to use both will cause Mocha to fail instead of simply ignoring `--grep`
+  - `--compilers` is no longer supported; attempting to use will cause Mocha to fail with a link to more information
+  - `-d` is no longer an alias for `--debug`; `-d` is currently ignored
+  - [#3275](https://github.com/mochajs/mocha/issues/3275): `--watch-extensions` no longer implies `js`; it must be explicitly added ([**@TheDancingCode**](https://github.com/TheDancingCode))
+- [#2908](https://github.com/mochajs/mocha/issues/2908): `tap` reporter emits error messages ([**@chrmod**](https://github.com/chrmod))
+- [#2819](https://github.com/mochajs/mocha/issues/2819): When conditionally skipping in a `before` hook, subsequent `before` hooks _and_ tests in nested suites are now skipped ([**@bannmoore**](https://github.com/bannmoore))
+- [#627](https://github.com/mochajs/mocha/issues/627): Emit filepath in "timeout exceeded" exceptions where applicable ([**@boneskull**](https://github.com/boneskull))
+- [#3556](https://github.com/mochajs/mocha/issues/3556): `lib/template.html` has moved to `lib/browser/template.html` ([**@boneskull**](https://github.com/boneskull))
+- [#2576](https://github.com/mochajs/mocha/issues/2576): An exception is now thrown if Mocha fails to parse or find a `mocha.opts` at a user-specified path ([**@plroebuck**](https://github.com/plroebuck))
+- [#3458](https://github.com/mochajs/mocha/issues/3458): Instantiating a `Base`-extending reporter without a `Runner` parameter will throw an exception ([**@craigtaub**](https://github.com/craigtaub))
+
+## :fax: Deprecations
+
+These are _soft_-deprecated, and will emit a warning upon use.  Support will be removed in (likely) the next major version of Mocha:
+
+- `-gc` users should use `--gc-global` instead
+- Consumers of the function exported by `bin/options` should now use the `loadMochaOpts` or `loadOptions` (preferred) functions exported by the `lib/cli/options` module
+
+Regarding the `Mocha` class constructor (from `lib/mocha`):
+
+- Use property `color: false` instead of `useColors: false`
+- Use property `timeout: false` instead of `enableTimeouts: false`
+
+All of the above deprecations were introduced by [#3556](https://github.com/mochajs/mocha/issues/3556).
+
+`mocha.opts` is now considered "legacy"; please prefer RC file or `package.json` over `mocha.opts`.
 
 ## :tada: Enhancements
 
-- [#3375]: Add support for comments in `mocha.opts` ([@plroebuck])
+Enhancements introduced in [#3556](https://github.com/mochajs/mocha/issues/3556):
+
+- Mocha now supports "RC" files in JS, JSON, YAML, or `package.json`-based (using `mocha` property) format
```

Comment:
> RC = **R**un**C**om ("run commands")

Historic artefact from [1960s-era](https://en.wikipedia.org/wiki/Run_commands) computing. That being said, it isn't stopping me from reading this as `Moch Arc`, but hey.

@cgalvarez, be glad I slacked off on rehauling [`atom-mocha`](https://github.com/Alhadis/Atom-Mocha): I was about to kill off support for `package.json` configs in favour of `mocha.opts`, now upstream is deprecated the latter in favour of the former. 😀

Reply:
True, "mocha.opts" is rcfile-like; the new ones are strictly configuration files.